crates/rspack_binding_api/src/rspack_resolver/mod.rs

```diff
@@ -213,6 +213,15 @@ impl ResolverFactory {
       symlinks: op.symlinks.unwrap_or(default.symlinks),
       builtin_modules: op.builtin_modules.unwrap_or(default.builtin_modules),
       enable_pnp: op.enable_pnp.unwrap_or_default(),
+      pnp_manifest: op.pnp_manifest.map(PathBuf::from).or_else(|| {
```

Comment:
as We discussed in rspack-resolver's PR. We just map to PathBuf::from is enough, no needs for default value.

Reply:
I have synchronized with napi in rspack-resolver